### DISCUSSION TITLE
UI Frameworks supported by `Nuxt 3`

### DISCUSSION THREAD

**Discussion by themodernpk** [nuxt/nuxt#9983] [General]
One of the best features that I enjoyed using in `Nuxt 2` was inbuilt support for multiple UI frameworks. Will this feature be back in `Nuxt 3` ? If yes what frameworks will be supported? 

I am especially interested to know about `oruga.io`.

Thanks

**Comment by atinux** [comment]
I think it depends also of how well these UI frameworks support Vue 3

**Reply by themodernpk** [reply]
@Atinux `oruga.io` does support Vue 3. They have a module for Nuxt 2 but not for Nuxt 3. `oruga-next` is on vue 3 https://oruga.io/documentation/#vue-3

I am unable to figure out any way to use any of the UI frameworks with Nuxt 3. I will be happy to develop a module or plugin for `oruga.io` and `buefy` if a guide or tutorial is provided.
